### PR TITLE
fix(DB/Creature): Elixir of Giants removed

### PR DESCRIPTION
**Pull request by baranzadeoglu**
Elixir of Giants removed from 5 tables

## Changes Proposed:
-  Delete Elixir of giants tables of the relevant 5 mobs
-  

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/1749 https://github.com/azerothcore/azerothcore-wotlk/issues/7911
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7911

## Tests Performed:

- 
- 

## How to Test the Changes:

1. query:
select creature_template.name, creature_loot_template.entry, creature_loot_template.chance from creature_loot_template join creature_template on creature_loot_template.Entry=creature_template.lootid where creature_loot_template.item="9206";

2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1631995181138245446.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631995181138245446');
+
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 5988) AND (`Item` IN (9206));
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 7154) AND (`Item` IN (9206));
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 7856) AND (`Item` IN (9206));
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 9240) AND (`Item` IN (9206));
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 10986) AND (`Item` IN (9206));
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1631995181138245446.sql:7]:
```suggestion
DELETE FROM `creature_loot_template` WHERE `Entry` IN (5988,7154,7856,9240,10986) AND (`Item` IN (9206));
```

**Review by Helias — APPROVED** [review]

**Comment by jamadaha** [comment]
Tested:
Ran query - Affected expected 5 rows
Checked whether the item then dropped from any mobs - Did not
Seems to work as expected.

**Comment by FrancescoBorzi** [comment]
what is the status of this PR?

**Comment by syssneck** [comment]
Looks good to me.

**Pull request merged**